src/components/crop-content/CropContent.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import ResizeDetectDiv from '../../core/hoc/ResizeDetectDiv';
+
+type CropContentProps = {
+  children?: React.ReactNode;
+  cropHeight?: string;
+  className?: string;
+  collapsed?: boolean;
+  style?: any;
+};
+
+export default class CropContent extends React.Component<CropContentProps> {
+  private containerElRef: HTMLDivElement;
+
+  public state = {
+    collapsed: this.props.collapsed === undefined ? true : this.props.collapsed,
```

Comment:
Ok actually I have a problem with the namings. I see that you have a prop `collapsed` and a state entry named the same. This is a bit confusing for me. It appears that you are only reading the prop when the component is mounted.
I think that I would not expect this, but if it is the behavior you want, you should call the prop `initiallyCollapsed` or something like that.

Reply:
Renamed